tests/S3/sig_hack.php

```diff
@@ -0,0 +1,10 @@
+<?php
+namespace Aws\S3;
+
+// Hack gmdate() to returned the canned result.
+function gmdate($format, $ts = null)
+{
+    return isset($_SERVER['aws_time'])
```

Comment:
Why not just require the file from the signature namespace and proxy to those functions, e.g., have `gmdate` in the `Aws\S3` namespace call `\Aws\Signature\gmdate`? That way, updates only need to occur in a single location.